packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -52,6 +52,32 @@ export interface AthenaStartQueryExecutionProps extends sfn.TaskStateBaseProps {
    * @default - No parameters
    */
   readonly executionParameters?: string[];
+
+  /**
+   * Specifies the query result reuse behavior for the query.
+   *
+   * @default - Query results are not reused
+   */
+  readonly resultReuseConfiguration?: ResultReuseConfiguration;
```

Comment:
```suggestion
  /**
   * Specifies, in minutes, the maximum age of a previous query result that Athena should consider for reuse.
   *
   * @default - Query results are not reused
   */
  readonly resultReuseConfigurationMaxAge?: Duration;
```
We could provide a cleaner interface, assuming that `ResultReuseConfiguration` is enabled if `resultReuseConfigurationMaxAge` is specified.
Also, we should use the more convenient `Duration` type for time intervals.